.github/actions/build-gk/action.yaml

```diff
@@ -0,0 +1,86 @@
+name: 'build gk'
+description: 'Build GenomeKit conda package and save it under ~/conda-bld'
+inputs:
+  platform:
+    description: 'platform name used in the mamba installer url'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: get meta.yaml from conda-forge
+      shell: bash -l -e {0}
+      run: |
+        set -x
+        mkdir -p conda-recipe
+        curl -s -L -o conda-recipe/meta.yaml https://raw.githubusercontent.com/conda-forge/genomekit-feedstock/main/recipe/meta.yaml
```

Comment:
actually now that i think about it, you need to SED the source so it builds from source rather than downloading a github tarball